### DISCUSSION TITLE
Plausible integration

### DISCUSSION THREAD

**Discussion by Nonolanlan1007** [coollabsio/coolify#2161] [Service Template Requests]
Hi!
Is it possible to add [Plausible](https://plausible.io/) to services list?
It's a self hosted alternative to Google Analytics 👀

**Comment by Nonolanlan1007** [comment]
This discussion is a duplicate of https://github.com/coollabsio/coolify/discussions/2160
I didn't create the first one in the good category :D

**Comment by endrekrohn** [comment]
Added plausible config in a PR #2473 👍

**Comment by andrasbacsai** [comment]
https://coolify.io/docs/resources/services/plausible#what-is-plausible-analytics
